models/workitem_repository_blackbox_test.go

```diff
@@ -0,0 +1,80 @@
+package models_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/almighty/almighty-core/app"
+	"github.com/almighty/almighty-core/criteria"
+	"github.com/almighty/almighty-core/models"
+	uuid "github.com/satori/go.uuid"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateLoadDeleteListWI(t *testing.T) {
+	doWithTransaction(t, func(ts *models.GormTransactionSupport) {
```

Comment:
A Test failure can be seen as a Bug Report, the more fine-grained the failure report is the easier it is to reason about and put you on the right track to fix it. 

In the current state a test failure says; 'something wrong with WI'.